vt.c: add missing include to fix building with uClibc                                                                                                    
                                                                                                                                                         
Backporting an upstream patch to fix a building issue with uClibc.                                                                                       
                                                                                                                                                         
Upstream commit:                                                                                                                                         
                                                                                                                                                         
  https://github.com/kernelslacker/trinity/commit/fb4a1adc4540f0702b84aa900f2b8ebed004885d                                                               
                                                                                                                                                         
Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com> 

--- trinity-1.5.orig/ioctls/vt.c	2015-03-02 17:08:20.000000000 +0100
+++ trinity-1.5/ioctls/vt.c	2015-03-20 18:57:15.000000000 +0100
@@ -1,6 +1,7 @@
 #include <sys/vt.h>
 #include <sys/ioctl.h>
 #include <linux/kd.h>
+#include <linux/serial.h>
 
 #include "utils.h"
 #include "ioctls.h"
